thinkshout/ts_motif
# Cleanup

# Pull Request

Closes Issue #12 

## Issue Description
See notes in #12 

## Summary of Changes
The existing version of ts_motif had a lot of base code that needed to be deleted with new builds, tokens that needed to be renamed and the style guide was a very long HTML doc. Base code in some of the css files has been simplified with Tailwind or removed, MOTIF graphics have been removed from css files, and the style guide has been refactored so it's a series of included templates that can be edited individually, rearranged and sorted to customize the style guide and client needs. 

The style guide has also been updated to include tiles of all color swatches. 

## Testing Instructions
Expect to pull in ts_motif on a new build, and have a starter style guide with minor improvements, and more intuitive token names for line height and z-index.
